holobot/framework/system/environment.py

```diff
@@ -4,17 +4,16 @@
 from holobot.sdk.system import IEnvironment
 from holobot.sdk.system.models import Version
 
-ROOT_PATH: str = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
-
 @injectable(IEnvironment)
 class Environment(IEnvironment):
     # NOTE: This version number is automatically updated on build by the script assign_version.yml.
```

Comment:
Since this file isn't modified by the pipeline anymore, I suggest you remove this now misleading comment. The same isn't required for the new file. Here it was important to have, to avoid accidentally modifying the file structure in a way that would have broken the generator.

If an explanation will be needed in the future, it should be added to the documentation then (because such a thing doesn't exist today).